clients/web/src/components/app/spaces/space.tsx

```
@@ -0,0 +1,37 @@
+import type { Space } from "@/types";
+import { Folder, EmptyFolder, Files } from "./folder";
+
+export default function Space({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Space'. Consider to delete it or rename it.

'Space' is defined here:


(lint/suspicious/noRedeclare)